content/applications/websites/website/configuration/address_autocomplete.rst

```diff
@@ -0,0 +1,110 @@
+====================
+Address Autocomplete
+====================
+
+To ensure that your customers' delivery addresses exist and are understood by the carrier,
+you can use Google Places API on your website.
+To do so, go to :guilabel:`Settings → SEO → enable “Address Autocomplete”`.
+
+
+.. image:: address_autocomplete/enable-address-autocomplete.png
+   :alt: Enable address autocomplete
+
+.. warning::
+   You need to generate your own API key, this one is only an example.
+
+But first, you need to generate a :guilabel:`Google Places API key`. This involves a few steps.
+The Places API allows developers to access detailed information about places using HTTP requests.
+Here's a short documentation guide on how to create your ID for the Google Places API:
+
+Step 1: Enable the Places API
+-----------------------------
+
+Go to the `Google Cloud Console <https://console.cloud.google.com/getting-started>`_:
+
+**Create a New Project:**
+If you don't have a project, create one by clicking on the project dropdown in the top right corner and selecting
+"New Project."
+Follow the prompts to set up your project.
+
+**Enable the Places API:**
+In the Cloud Console, go to the `APIs & Services > Dashboard <https://console.cloud.google.com/apis/dashboard>`_
+and click on "+ ENABLE APIS AND SERVICES."
+Search for :guilabel:`"Places API"` and select it. Click on the :guilabel:`"Enable"` button.
+
+Step 2: Create API Credentials
+------------------------------
+
+In the Cloud Console, go to `APIs & Services > Credentials <https://console.cloud.google.com/apis/credentials>`_.
+
+**Create Credentials:**
+Click on the :guilabel:`"Create Credentials"` button and select :guilabel:`"API key."`
+A pop-up appears with your API key.
+
+**Restrict the API Key (Optional):**
+
+.. note::
+   For security purposes, you can restrict `the usage of your API key.
+   Under the :guilabel:`"API restrictions"` section, you can specify which APIs your key can access.
+   For the Places API, you can restrict it to only allow requests from specific websites or apps.
+
+.. important::
+   Save Your API Key: Copy your API key and securely store it.
+   Do not share it publicly or expose it in client-side code.
+
+Step 3: Usage
+-------------
+
+Now that you have your API key, you can start using it in your applications to make requests to the Google Places API.
+Include the API key in your requests using the key parameter.
+
+.. important::
```

Comment:
I'm not sure it makes sens to mention this info in Odoo documentation. We can discuss it :)